circle.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+workflows:
+  version: 2
+  linux-builds:
+    jobs:
+      - build-x86_64
+      - build-i686
+
+jobs:
+  build-x86_64:
+    docker:
+#      - image: ubuntu:14.04
+      - image: circleci/python:2.7
```

Comment:
the ideal configuration for ci would probably be oldest supported versions of both glibc and compiler, but the closest thing to that combination that doesn't require building our own gcc would be to use the devtoolset, and that has its own issues. I was trying to avoid needing to build a custom image for this. I'd have used the bare minimum ubuntu 14.04 image except for the hassle of adding a non-root user, which the circle image already did - it's also already locally fetched so doesn't need to download any large new containers to start the build. getting things from apt-get is fast enough to not need to optimize.